### PR TITLE
Add seqkit stats and fx2tab tools

### PR DESCRIPTION
**Pull request by gregvonkuster**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/seqkit/macros.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<macros>
+    <token name="@TOOL_VERSION@">2.2.0</token>
+    <token name="@VERSION_SUFFIX@">0</token>
+    <token name="@PROFILE@">20.09</token>
+    <xml name="bio_tools">
+        <xrefs>
+            <xref type="bio.tools">megan</xref>
```
Comment (bgruening) [inline tools/seqkit/macros.xml:7]: That looks wrong :-)

File: tools/seqkit/seqkit_fx2tab.xml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+$header_line
+$length
+$name
+$no_qual
+$only_id
+#if str($input.ext) in ['fastqsanger', 'fastqsanger.gz']:
+    --qual-ascii-base 33
+#end if
+$seq_hash
+> $output
```
Comment (bgruening) [inline tools/seqkit/seqkit_fx2tab.xml:29]: Quotes

File: tools/seqkit/seqkit_stats.xml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+seqkit stats 
+'${input_identifier}'
+$all
+$basename
+#if str($input.ext) in ['fastqsanger', 'fastqsanger.gz']:
+    --fq-encoding sanger
+#end if
+$skip_err
+$tabular
+> $output
```
Comment (bgruening) [inline tools/seqkit/seqkit_stats.xml:23]: Quotes

File: tools/seqkit/seqkit_stats.xml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        <test expect_num_outputs="1">
+            <param name="input" value="input1.fasta.gz" ftype="fasta.gz"/>
+            <param name="all" value="--all"/>
+            <output name="output" file="stats_output2.tabular" ftype="tabular"/>
+        </test>
+    </tests>
+    <help>
+**What it does**
+
+Outputs simple statistics of FASTA/Q files.
```
Comment (bgruening) [inline tools/seqkit/seqkit_stats.xml:54]: For example? What kind of stats?

File: tools/seqkit/macros.xml (hunk 1 above)
Comment (gregvonkuster) [reply tools/seqkit/macros.xml]: Fixed - thanks!

File: tools/seqkit/seqkit_fx2tab.xml (hunk 2 above)
Comment (gregvonkuster) [reply tools/seqkit/seqkit_fx2tab.xml]: Fixed - thanks!

File: tools/seqkit/seqkit_stats.xml (hunk 3 above)
Comment (gregvonkuster) [reply tools/seqkit/seqkit_stats.xml]: Fixed - thanks!

File: tools/seqkit/seqkit_stats.xml (hunk 4 above)
Comment (gregvonkuster) [reply tools/seqkit/seqkit_stats.xml]: Fixed - thanks!

**Comment by gregvonkuster** [comment]
Sorry about these errors @bgruening - they should now be fixed.

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Merci!

**Pull request merged**